resources/js/visual-editor/site-editor/entity-editor-canvas.css

```
@@ -0,0 +1,140 @@
+.ap-site-editor__entity-canvas-surface .block-editor-block-list__layout {
+    min-height: 300px;
+}
+
+.ap-site-editor__sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)